### ISSUE TITLE
Nice effort guys!

### ISSUE THREAD

**Issue by viljaste** [Wodby/docker4drupal#10]
Hey,

Have you considered supporting almost a year old [Dockerized Drupal](https://dockerizedrupal.com/) initiative (FYI I'm the initial author of that project) as this project is also trying to make Drupal development with Docker easier. I think that we are trying to achieve the same goal here, but correct me if I'm wrong.

**Comment by csandanov** [comment]
Thanks for the link, I personally didn't know about this initiative, we'll definitely take a look at your project. The main reason we started this initiative is that docker has finally launched a native app for mac/windows, so we decided it might be a good opportunity to share our [bundles](https://docs.wodby.com/infrastructure/bundles/) with the community.

**Comment by macmladen** [comment]
I took a look at  **Dockerized Drupal** initiative and it looks interesting as the idea of having isolated local development is the way to go.

The main difference is in virtualization technology: **Dockerized Drupal** is using older Docker method (toolbox) while **docker4drupal** is using more modern **Docker for Mac** application.

Vagrant - virtual box works for older systems (pre Intel core processors before 2010) and is still the only solution for those and systems not running 10.10.3 [ref: Docker docs](https://docs.docker.com/docker-for-mac/)

Since Mac machines that are newer than 2010 and have 10.10.3 or newer OS version have hypervisor they do not need vagrant/virtualbox as an additional virtualization layer (sure, there is still a long way to stabilize both native Docker clients for Mac and Windows but I think it is preferred way to go) so they are more lightweight than old toolbox solution.

Other than that, on a container level there probably could be some common ground.

**Comment by macmladen** [comment]
@viljaste I found your documentation quite confusing. Not that I personally could not find the way to make that work but it could be improved (streamlined) considerably and that would help adoption.

Current appearance (style) is also very hard to follow so I'd suggest cleaning style or using some other system like [Git Book](https://www.gitbook.com/) or [Read The Docs](https://readthedocs.org/) like **Drush**, even plain **giihub** pages or wiki offer nice, easy to follow appearance and structure.

But technology stack should also be improved to current ones.

**Comment by PavelPrischepa** [comment]
I'm going to close the issue due to no activity.
